openbrews/recipe/detail-edit/edit-recipe.html

```diff
@@ -50,6 +50,39 @@
                         <input type="text" name="mashEfficiency" ng-model="recipe.mashEfficiency" class="text-right">
                         <span class="input-addon">%</span>
                     </label>
+                    <label for="steepEfficiency" class="item item-input item-input-group">
+                        Steep Efficiency
+                        <input type="text" name="steepEfficiency"
+                        ng-model="recipe.steepEfficiency" class="text-right">
+                        <span class="input-addon">%</span>
+                    </label>
+
+                    <label for="og" class="item item-input">
+                        OG
```

Comment:
Yes, I did intend that. Sometimes calculators just can't account for all the possible things that could influence these values. So, if a user has a better calculated value, or measures a different value when making the recipe, they can enter that in here